manifest.json

```diff
@@ -1,20 +1,25 @@
 {
     "manifest_version": 2,
-
     "name": "Cowin Booking",
     "description": "automatically fill data on cowin portal to speed up covid vaccine slot booking",
     "version": "2.0.6",
     "content_scripts": [
         {
-            "matches": ["https://selfregistration.cowin.gov.in/*"],
+            "matches": [
+                "https://selfregistration.cowin.gov.in/*"
+            ],
             "js": [
                 "jquery.min.js",
                 "contentScript.js",
-                "shortcut.js"
+                "shortcut.js",
+                "bootstrap.min.js"
```

Comment:
move this to top, just after jquery.
Also did you check if this works (including css)?

Reply:
Can you just confirm the order of the js array